config/constants/settings/definitions.rb

```diff
@@ -497,42 +552,40 @@
   add :journal_aggregation_time_minutes,
       default: 5
 
-  # Allow override of LDAP options
   add :ldap_force_no_page,
+      description: 'Force LDAP to respond as a single page, in case paged responses do not work with your server.',
       format: :string,
       default: nil,
       writable: false
 
-  # Allow users to manually sync groups in a different way
-  # than the provided job using their own cron
   add :ldap_groups_disable_sync_job,
+      description: 'Deactivate regular synchronization job for groups in case scheduled as a separate cronjob',
       default: false,
       writable: false
 
   add :ldap_users_disable_sync_job,
+      description: 'Deactive user attributes synchronization from LDAP',
       default: false,
       writable: false
 
-  # Update users' status through the synchronization job
   add :ldap_users_sync_status,
+      description: 'Enable user status (locked/unlocked) synchronization from LDAP',
       format: :boolean,
       default: false,
       writable: false
 
-  add :ldap_tls_options,
-      format: :hash,
-      default: {},
-      writable: true
```

Comment:
We still have this in our `configuration.yml.example`. Not sure if it still applies.